Add basic dragonfly support.

craigmcnamara opened the pull request:
Allows cropping of images managed by dragonfly.

Commit:
Add basic dragonfly support.

Ricardonacif commented:
Awesome! Thanks for that @craigmcnamara. Can you take a quick look on the specs? Looks like they're failing

Commit:
Add dragonfly as a development dependency.

craigmcnamara commented:
Hey, thanks for looking so quick. I'll get this into shape to merge.

Commit:
`Gemfile.lock` files make it difficult to test on Travis.

I’ll try to get appraisal setup so we can test across versions of gems and rails that are important.

Commit:
Drop the bundle args

Commit:
Update config so tests can run properly.

Commit:
Add the manifest to the rails app so assets build in CI

Commit:
Don’t use the master branch of sass-rails

jaw6054 commented on Gemfile.lock:
```diff
@@ -1,242 +0,0 @@
-GIT
-  remote: git://github.com/gregbell/active_admin.git
-  revision: 21f76b1faf651cd4f71dd24997102f7eb83b49d2
```
active_admin_jcrop.gemspec
